app/controllers/application_controller.rb

```diff
@@ -24,7 +24,7 @@ def configure_permitted_parameters
   end
 
   helper_method memoize def topup_utc_time
```

Comment:
btw, correct name for this method is `topup_time`

The "utc" part is a technical detail and should be avoided.